servers/quarkus-server/src/test/java/org/projectnessie/server/authn/AuthenticationDisabledProfile.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (C) 2020 Dremio
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.projectnessie.server.authn;
+
+import com.google.common.collect.ImmutableMap;
+import io.quarkus.test.junit.QuarkusTestProfile;
+import java.util.Map;
+
+/** A simple {@link QuarkusTestProfile} that disabled Nessie authentication. */
+public class AuthenticationDisabledProfile implements QuarkusTestProfile {
```

Comment:
One thing I noticed w/ Quarkus test profiles:
You need actually two profiles: one for "normal" unit tests and one when running the tests against a native image. The latter needs
```
  @Override
  public String getConfigProfile() {
    return "prod";
  }
```

Reply:
Do you think we should have separate tests for auth in native mode? I assumed it was not necessary since the logic is contained in a basic java class.

Reply:
I've added `ITNativeOpenIdAuthentication` in 739575aeb2cbc3975b57cb11b1eda4127434ed5a. I guess it is not necessary to run native tests for basic auth since it is only meant to be supported for unit tests. WDYT?

Reply:
Yea - that's okay I think.